LCM/Code/App/task.h

```diff
@@ -14,7 +14,7 @@ typedef enum
 } CELL_TYPE;
 
 //#define PINTV
-#define XRV
+//#define XRV
```

Comment:
There is no need for any of these, `LCM_Light_Control_IO_WS2812_New.csolution.yml`